pkg/doc/verifiable/credential.go

```diff
@@ -561,21 +687,26 @@ func describeSchemaValidationError(result *gojsonschema.Result) string {
 	return errMsg
 }
 
-func getCredentialSchema(schema *CredentialSchema, opts *credentialOpts) (gojsonschema.JSONLoader, error) {
-	schemaLoader := defaultSchemaLoader
-	if schema != nil && !opts.disabledCustomSchema {
+func getSchemaLoader(schemas []CredentialSchema, opts *credentialOpts) (gojsonschema.JSONLoader, error) {
```

Comment:
Just double checking that it was intentional to remove the pointer type.
i.e., you have []CredentialSchema rather than []*CredentialSchema.

Reply:
Sure, Troy. Previously, `getCredentialSchema()` got Schema Loader for single credential schema, now it gets the same for the first found appropriate schema. So it's correct here.

Reply:
Sure - agree that a slice is needed. I was more curious about the pointer type inside.